docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: (2015-10-29)
+- RFC PR: (leave this empty)
+- AMS Issue: (leave this empty)
+
+# Summary
+
+Provide a consistent API for the user of the AMS.
+
+# Motivation
+
+The actual public API is defined under `ActiveModelSerializers`,
+`ActiveModel::Serializer` and `ActiveModel`.
```

Comment:
@bf4 I think here you go beyond the namespace issue I am right? You mentioned things like `ActiveModelSerializers::V08` and `ActiveModelSerializers::Current::Store` that I think are new idea of implementations. 

The Idea of RFC is to define a consistent namespace I will try to focus on it.

At the column desired we have two namespaces as public API `ActiveModelSerializers` and `ActiveModel`? In your opinion does not make sense a single namespace?

Thanks for the feedback